packages/contracts-bedrock/contracts/universal/FeeVault.sol

```diff
@@ -40,12 +47,28 @@ abstract contract FeeVault {
     uint256 public totalProcessed;
 
     /**
-     * @param _recipient           Wallet that will receive the fees on L1.
-     * @param _minWithdrawalAmount Minimum balance before a withdrawal can be triggered.
+     * @notice Emitted each time a withdrawal occurs.
+     *
+     * @param value             Amount that was withdrawn (in wei).
+     * @param to                Address that the funds were sent to.
+     * @param from              Address that triggered the withdrawal.
+     * @param withdrawalNetwork Network which the to address will receive funds on.
+     */
+    event Withdrawal(uint256 value, address to, address from, WithdrawalNetwork withdrawalNetwork);
```

Comment:
We should have 2 withdrawal events to stay backwards compatible, then we can announce the sunset of the old one and remove it after some time. This will prevent existing dashboards from breaking, so that they have time to upgrade